### ISSUE TITLE
Chromium based browsers cannot drag and drop tabs across monitors

### ISSUE THREAD

**Issue by AlexAegis** [baskerville/bspwm#1088]
Firefox works just fine.

Two browser windows, one on each monitor, multiple tabs opened on both. When dragging a tab to another, on chromium browsers it just does nothing. In firefox it does.
Within a single monitor it works.

bspwm version: 0.9.7-10-g2ffd9c1
chromium version: chromium-80.0.3987.163
brave version: Brave Browser 80.1.5.122
firefox version: Mozilla Firefox 74.0.1
(freshest everything on arch)

**Comment by jameh** [comment]
fwiw, it is also impossible to drag a chromium tab at all across the monitor border, i.e. to move it to a new window on the other monitor. (which works for firefox)

I have also noticed that the cursor changes to a special drag-and-drop cursor when dragging firefox tabs whereas it doesn't for chromium tabs:

![cursor](https://user-images.githubusercontent.com/1538657/81448123-be943e00-9132-11ea-821b-c11a0973a3c4.png)

**Comment by fuzzy-focus** [comment]
Seems to still not be possible.

**Comment by zjeffer** [comment]
[2022-08-25 13-49-47.webm](https://user-images.githubusercontent.com/4633209/186657001-689c6ea6-0f2d-46e9-9d2d-1ef6c124b136.webm)

Seems to work for me, but I do have the same problem as @sidequestboy: a new window is created even without releasing the mouse. 

@fuzzy-focus Does it not work at all, even when dragging the mouse all the way up to the top bar?

**Comment by fuzzy-focus** [comment]
I'm running bspwm 0.9.10-2 and for me it looks like this:

https://user-images.githubusercontent.com/13691063/186711397-3213985e-d1ed-4e6f-9f5e-603691f4b0a0.mp4

This is only for a single screen, but I think it is related.

**Comment by zjeffer** [comment]
Is that Google Chromium, or another Chromium-based browser?

**Comment by fuzzy-focus** [comment]
It's google-chrome-stable from AUR.

**Comment by zjeffer** [comment]
google-chrome-stable works for me as well. Maybe one of your extensions is creating the problem?

**Comment by fuzzy-focus** [comment]
Nope, not an extension. Just tested it in Incognito mode, same thing.

I wonder if the problem is with my bspwm settings?

``` 
bspc config window_gap          10

bspc config single_monocle        true
bspc config borderless_monocle    true
bspc config gapless_monocle       true
bspc config pointer_follows_focus true
bspc config focus_follows_pointer true
``` 
I suspect that especially the last two might be causing the fun.
